sigle/src/modules/publicHome/components/PublicHome.tsx

```diff
@@ -182,6 +186,23 @@ export const PublicHome = ({ file, settings, userInfo }: PublicHomeProps) => {
     );
   };
 
+  const encodeImageUrl = (url: string | undefined) => {
+    if (!url) {
+      return;
+    }
+
+    // encode part of url that includes file name which may contain special characters
+    const split = 'settings/';
```

Comment:
@gregogun why is this logic needed? `encodeURI` should not update the non special chars no?